loops.js

```diff
@@ -51,3 +65,18 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++)
+    if (i <= 2) {
+        console.log(i);
+    }
+
+    if (i >= 3) {
+        console.log("> " + i);
+    }
+
+    if (i >= 6) {
+        console.log(">> " + i);
+    }
+
+//Op bovenstaande bonusopdracht liep ik helemaal vast?? Is helaas niet gelukt - error ///
```

Comment:
Helemaal niet slecht hoor.
Om de error op te lossen, had je alleen de accolades van je for loop toe hoeven voegen.
for() **{}**.  :)

Daarna doet je loop je het , alleen kloppen de uitkomsten nog niet.
De eerste if klopte.
De 2e en 3e if's gaan allebei steeds af.

Omdat groter dan 6 ook altijd groter dan 3 is.
Daarom worden allebei de console.logs uitgevoerd.

In feite wil je dit 

0 tot 2 geeft i 
3 tot 5 geeft > i    (hier zou je kunnen zeggen i <= 5, want je had al gecheckt op <= 2 dus dat is veilig)
6+ geeft  >> i
Ook voorkom je dat beide if's uitgevoerd worden door er een else if van te maken ipv een if.
Alles boven de 6 hoef je dan niet meer te checken want dat wordt afgevangen in de else {}

Dus 
```
if(i <= 2) {
 console.log(i);
}
else if(i <= 5) {
 console.log("> " + i);
}
else {
 console.log(">> " + i);
}
```

Reply:
ahh super, dankjewel! ik snap hem nu inderdaad :)